src/pages/workspace/WorkspaceInviteMessagePage.tsx

```diff
@@ -100,14 +86,22 @@ function WorkspaceInviteMessagePage({
         // eslint-disable-next-line react-compiler/react-compiler, react-hooks/exhaustive-deps
     }, []);
 
+    useEffect(() => {
+        if (isEmptyObject(invitedEmailsToAccountIDsDraft)) {
```

Comment:
We needed an early return here for cases where the `welcomeNote` is intentionally not set. This is to avoid default value being set for such cases. This caused #49899.